cmodel/predefined/_sir_model.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+
+from cmodel.model import CompartmentalModel
+
+
+class SIR_model(CompartmentalModel):
```

Comment:
Change class name to one similar to ``cmodel.predefined._seirv_model.ModelSEIRV``

```suggestion
class ModelSIR(CompartmentalModel):
```